### PR TITLE
Remove unsupported Node.js LTS versions

### LINKED ISSUE #56: Remove support for unsupported Node versions
This repo should only test against supported LTS versions.

### PR DESCRIPTION
**Pull request by snacqs**
Hey @jsumners! This PR closes #56 🙂

### CODE REVIEW

**Comment by snacqs** [comment]
@delvedor right now `engines` field is missing package.json. Should I add it?

**Comment by jsumners** [comment]
@delvedor why? It ultimately just ends up causing noise that people ignore when installing modules.

The only time I have found it useful is when adding it to an application's `package.json` and deploying to Heroku. Heroku will use the root `package.json`'s engines block to determine which version of Node to use for the deployment. The dependency engines are just ignored.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '12', '10', '8', '6' ]
+        node: [ '12', '10' ]
```
Comment (jsumners) [inline .github/workflows/ci.yml]:
```suggestion
        node: [ '13', '12', '10' ]
```

**Comment by jsumners** [comment]
Merging. Another PR can be used for the engines thing if it is determined to be truly necessary.

**Pull request merged**